traefik.go

```diff
@@ -1,16 +1,130 @@
 package main
 
 import (
+	"encoding/json"
+	log "github.com/Sirupsen/logrus"
+	"github.com/containous/flaeg"
+	"github.com/containous/staert"
+	"github.com/containous/traefik/middlewares"
+	"github.com/containous/traefik/provider"
 	fmtlog "log"
+	"net/http"
 	"os"
+	"reflect"
 	"runtime"
+	"strings"
 )
 
 func main() {
 	runtime.GOMAXPROCS(runtime.NumCPU())
-	if err := traefikCmd.Execute(); err != nil {
+
+	//traefik config inits
+	traefikConfiguration := NewTraefikConfiguration()
+	traefikPointersConfiguration := NewTraefikPointersConfiguration()
```

Comment:
I need a little bit more *context* on that (it's a design decision from `flaeg` it seems, but I wonder why it is needed :stuck_out_tongue_closed_eyes:)